netlink-packet-route/src/rtnl/link/nlas/mod.rs

```diff
@@ -500,7 +501,15 @@ impl<'a, T: AsRef<[u8]> + ?Sized> ParseableParametrized<NlaBuffer<&'a T>, u16> f
                     }
                     AfSpecInet(nlas)
                 }
-                AF_BRIDGE => AfSpecBridge(payload.to_vec()),
+                AF_BRIDGE => {
+                    let mut nlas = vec![];
+                    let err = "invalid IFLA_AF_BRIDGE value";
+                    for nla in NlasIterator::new(payload) {
+                        let nla = nla.context(err)?;
+                        nlas.push(af_spec_bridge::AfSpecBridge::parse(&nla).context(err)?);
+                    }
+                    AfSpecBridge(nlas)
```

Comment:
The process might be wrong. The kernel code indicate a interface could have mulitple family's `IFLA_AF_SPEC`. Let me do some digging.

Reply:
My fault. The kernel has `rtnl_register(PF_BRIDGE, RTM_GETLINK, NULL, rtnl_bridge_getlink, 0);` which indicate kernel is not using two layer nested NLA for `PF_BRIDGE`